lib/rules/index.cjs

```diff
@@ -1,259 +1,396 @@
 'use strict';
 
-const node_module = require('node:module');
-const importLazy = require('import-lazy');
+const _interopNamespaceDefaultOnly = e => Object.freeze({ __proto__: null, default: e });
 
-var _documentCurrentScript = typeof document !== 'undefined' ? document.currentScript : null;
-// @ts-expect-error -- TS1343: The 'import.meta' meta-property is only allowed when the '--module' option is 'es2020', 'es2022', 'esnext', 'system', 'node16', or 'nodenext'.
-const require$1 = node_module.createRequire((typeof document === 'undefined' ? require('u' + 'rl').pathToFileURL(__filename).href : (_documentCurrentScript && _documentCurrentScript.src || new URL('lib/rules/index.cjs', document.baseURI).href)));
-
-/** @type {import('stylelint')['rules']} */
+/** @type {import('stylelint').BuiltInRules} */
 const rules = {
```

Comment:
refactor
suggestion

Maybe create an array with all the rules' names and then loop over it to add the getters?